app/src/main/res/layout/activity_main.xml

```diff
@@ -1,21 +1,41 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.appcompat.widget.LinearLayoutCompat
+    android:orientation="vertical"
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
+    <ImageView
+        android:id="@+id/checkInternetConnectionsIV"
+        android:layout_width="110dp"
+        android:layout_height="100dp"
+        android:layout_marginTop="180dp"
+        android:layout_gravity="center"
+        android:contentDescription="@string/todo"
+        app:srcCompat="@drawable/broadcast" />
+
+    <TextView
+        android:id="@+id/checkInternetConnectionsTV"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:layout_marginTop="18dp"
+        android:layout_marginBottom="70dp"
+        android:fontFamily="sans-serif-black"
+        android:text="@string/CheckNet"
+        android:textColor="@color/black"
+        android:textSize="20sp"
+        app:layout_constraintTop_toBottomOf="@+id/imageView3"/>
+
     <Button
-        android:id="@+id/retry"
+        android:layout_marginLeft="20dp"
```

Comment:
two attributes `marginLeft/marginRight,` having the same value passed in, can be replaced with the` marginHorizontal`

and for the future (not for this time) try to avoid attributes containing "left/right" wording and use rather their "start/end" counterparts. 

i.e : "marginStart" instead of "marginLeft" and "marginEnd" instead of "marginRight" 
https://developer.android.com/about/versions/android-4.2#RTL